tests/integration/replication.tcl

```diff
@@ -78,6 +78,7 @@ start_server {tags {"repl"}} {
         }
 
         test {BRPOPLPUSH replication, when blocking against empty list} {
+            r config resetstat
```

Comment:
`r` (the last server spawned) is the master, we want to look into the command stats of the replica.
```suggestion
            $A config resetstat
```